spec/openapi.yaml

```diff
@@ -11106,7 +11106,7 @@ components:
                             the hashcode for a string via String API.
                           type: string
                         localeId:
-                          description: An identifying code for a language in Smartling.
+                          description: An identifying code for a language in Smartling (it is targetLocaleId).
```

Comment:
An identifying code for a target language in Smartling.

Reply:
Done